Build Image Release Artefacts 3.0.yml

```diff
@@ -0,0 +1,1634 @@
+name: Build Image Release Artefacts 3.0-$(Build.BuildId)
+
+variables:
+  - group: "External IP Addresses"
+  - group: "ImageVersion"
+  - group: "Logins"
+  - name: JpnLanguage
+    value: JPN
+  - name: Windows-2016-AMI
+    value: Windows_Server-2016-English-Full-SQL_2017_Express*
+  - name: Windows-2016-AMI-JPN
+    value: Windows_Server-2016-Japanese-Full-Base*
+  - name: Windows-2019-AMI
+    value: Windows_Server-2019-English-Full-SQL_2019_Express*
+  - name: Windows-2019-AMI-JPN
+    value: Windows_Server-2019-Japanese-Full-Base*
+  - name: Windows-2022-AMI
+    value: Windows_Server-2022-English-Full-SQL_2019_Express*
+  - name: Windows-2022-AMI-JPN
+    value: Windows_Server-2022-Japanese-Full-Base*
```

Comment:
All the Windows_nnnn variables above are shared with "AWS Images - Cookbooks CI 3.0".
So get Nikhil or Tarun to create a Variable Group called "AWS base AMI Image Names" and add all of these variables to that. Please inform Divinash of the same.
@nkhl-accolite

Reply:
removed windows_nnnn variables from yaml & added in variable group and that is mentioned in yaml file.

Reply:
i have removed Windows_nnnn variables from yaml file & added in mentioned variable group and mentioned over yaml as well